docs/Reference/Hooks.md

```diff
@@ -412,6 +414,23 @@ fastify.addHook('onReady', async function () {
   await loadCacheFromDatabase()
 })
 ```
+### onListen
+
+Triggered after the server starts listening for requests.
```

Comment:
The documentation will also require changes 👍

Reply:
```suggestion
Triggered when the server starts listening for requests.
```